### PR TITLE
Preperation for http/1.x and twig 2.x

### PR DESCRIPTION
**Pull request by ghost**
- Modify used prefix for twig extension.
- Remove (PhpStorm) user configuration from repository.
- Preperation for http/1.x and twig 2.x support.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# EditorConfig is awesome: http://EditorConfig.org
```
Comment (b3nl) [inline .editorconfig:1]: Why?

File: src/BestItOnDemandAssetsBundle.php [hunk 2]
```diff
@@ -5,7 +5,8 @@
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 /**
- * Class BhAppBundle
+ * Class BestItOnDemandAssetsBundle
```
Comment (b3nl) [inline src/BestItOnDemandAssetsBundle.php:8]: Thx

File: src/DependencyInjection/BestItOnDemandAssetsExtension.php [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 
+use Exception;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * Loads the config for the bundle.
+ *
```
Comment (b3nl) [inline src/DependencyInjection/BestItOnDemandAssetsExtension.php:13]: Why? https://phpdoc.org/docs/latest/guides/docblocks.html#summary

File: src/DependencyInjection/BestItOnDemandAssetsExtension.php [hunk 4]
```diff
@@ -18,13 +20,22 @@ class BestItOnDemandAssetsExtension extends Extension
 {
     /**
      * Loads the bundle config.
+     *
```
Comment (b3nl) [inline src/DependencyInjection/BestItOnDemandAssetsExtension.php:23]: Why? https://phpdoc.org/docs/latest/guides/docblocks.html#summary

Comment (b3nl) [reply src/DependencyInjection/BestItOnDemandAssetsExtension.php:23]: Or "The description can be as long as you would like and ends when a tag is encountered for the first time." -> https://phpdoc.org/docs/latest/guides/docblocks.html#description

File: src/DependencyInjection/BestItOnDemandAssetsExtension.php [hunk 5]
```diff
@@ -18,13 +20,22 @@ class BestItOnDemandAssetsExtension extends Extension
...
      * @param array $configs
      * @param ContainerBuilder $container
+     *
```
Comment (b3nl) [inline src/DependencyInjection/BestItOnDemandAssetsExtension.php:26]: Handy work or phpstorm automatism?

File: src/DependencyInjection/Configuration.php [hunk 6]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace BestIt\OnDemandAssetsBundle\DependencyInjection;
+
+use RuntimeException;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * Class Configuration
```
Comment (b3nl) [inline src/DependencyInjection/Configuration.php:10]: So "Configuration for BEstItOnDemandAssBundle is the summary or description?

File: src/Twig/SimpleFunction/CallbackTrait.php [hunk 7]
```diff
@@ -15,38 +16,53 @@
...
     protected $allowedFileEnding = '';
 
     /**
      * The asset extension.
+     *
      * @var AssetExtension
      */
-    protected $assetExtension = null;
+    protected $assetExtension;
+
```
Comment (b3nl) [inline src/Twig/SimpleFunction/CallbackTrait.php:30]: Sometimes no new lines and sometimes not?

File: src/Twig/AssetHelperExtension.php [hunk 8]
```diff
@@ -24,21 +22,37 @@ class AssetHelperExtension extends Twig_Extension
 {
     /**
      * The asset extension.
+     *
      * @var AssetExtension
      */
-    protected $assetExtension = null;
+    protected $assetExtension;
```
Comment (b3nl) [inline src/Twig/AssetHelperExtension.php:28]: Sometimes new lines and sometimes not?

**Review by b3nl — APPROVED** [review]

File: src/DependencyInjection/BestItOnDemandAssetsExtension.php (hunk 3 above)
Comment (ghost) [reply src/DependencyInjection/BestItOnDemandAssetsExtension.php:13]: Autoformat by PhpStorm, when selected PSR

File: src/DependencyInjection/BestItOnDemandAssetsExtension.php (hunk 4 above)
Comment (ghost) [reply src/DependencyInjection/BestItOnDemandAssetsExtension.php:23]: Autoformat by PhpStorm, when selected PSR

File: src/DependencyInjection/Configuration.php (hunk 6 above)
Comment (ghost) [reply src/DependencyInjection/Configuration.php:10]: Desc.

**Pull request merged**